### DISCUSSION TITLE
tls client mode without cert

### DISCUSSION THREAD

**Discussion by manojmth** [hnimminh/libresbc#96] [Q&A]
Hi, thanks for this software, haven't tried it out yet. Have a use case as follows which Asterisk cannot handle, wish to know if libresbc can be used instead
Should act as a tls sip client like microsip using peer cert only without presenting its own for both incoming and outgoing comm to a certain external peer
Provide digest auth to peer for own authentication
Modify headers including contact header for register, invite etc

Thanks.

**Comment by hnimminh** [comment]
libresbc will able to cover all case above
